webapp/channels/src/components/post_view/post_image/post_image.tsx

```diff
@@ -14,7 +17,7 @@ import {ModalIdentifiers} from 'utils/constants';
 
 import type {ModalData} from 'types/actions';
 
-interface Props {
+type Props = {
```

Comment:
why did we change from interface to type?

Reply:
I think it's because VS Code complains when `Props` is an unexported interface.

![Screenshot 2024-03-18 at 2 04 23 PM](https://github.com/mattermost/mattermost/assets/3277310/be2e4f8e-c8d0-4468-bb9c-f16ee234f5e8)

I've been seen that a lot, but since it doesn't cause any issues when running the app or with `make check-types`, I haven't bothered doing a widespread fix for it. I have been leaning more towards changing `Props` to be a type instead of exporting the interface, but I don't have a good reason for it other than it being slightly shorter to type `type Props = {` instead of `export interface Props {`

Reply:
yes, that was the reason. And I saw larkox's example, so I decided to do the same. 
https://github.com/mattermost/mattermost/pull/24187/files#r1288140965

What should I do then?

Reply:
i think what you have is fine as per @hmhealey comment